src/summary/history/allergy-card-level-three.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import style from "./allergy-card-level-three.css";
+
+export function AllergyCardLevelThreeAdd(props: AllergyCardLevelThreeAddProps) {
+  return (
+    <SummaryCard
+      name="Add Allergy"
+      link=""
+      match={props.match}
+      styles={{
+        width: "100%",
+        background: "var(--omrs-color-bg-medium-contrast)"
+      }}
+    >
+      <div>
+        <h4 className={style.allergyHeader}>Category of reaction</h4>
```

Comment:
Should this be within <form> tags or is that just in the case where we are submitting data? (in this case, this will primarily drive which components to display) @joeldenning ?

Reply:
I spoke to Joel. Any input should be enclosed within a form tag.

Reply:
Wrapper the inputs in form tag